modules/manage/pages/iceberg/iceberg-topics-databricks-unity.adoc

```diff
@@ -160,6 +169,8 @@ rpk topic create <topic-name> --topic-config=redpanda.iceberg.mode=key_value
 ----
 rpk topic alter-config <topic-name> --set redpanda.iceberg.mode=key_value
 ----
++
+Th `key_value` mode creates an Iceberg table for the topic consisting of two columns, one for the record metadata including the key, and another binary column for the record's value. See xref:manage:iceberg/choose-iceberg-mode.adoc[] for more details on Iceberg modes. 
```

Comment:
Typo in the first word. Since you'll be fixing the typo anyway, consider moving this above the `Create a new topic and set redpanda.iceberg.mode` section. Describe it before it is used.